src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    private static final int MAX_VALUE_FIGURES = 6;
+
+    public static void main(String[] args) {
+
+        final Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier(random);
+        FigureSupplier figureSupplier = new FigureSupplier(random, colorSupplier);
+
+        Figures[] figures = new Figures[MAX_VALUE_FIGURES];
```

Comment:
do we need this array?

Reply:
![image](https://user-images.githubusercontent.com/115244741/217843617-77510e9d-0769-4254-9b15-699345bd7d8e.png)